examples/run_glue_tpu.py

```diff
@@ -341,6 +355,9 @@ def main(args):
     label_list = processor.get_labels()
     num_labels = len(label_list)
 
+    if not xm.is_master_ordinal():
+        xm.rendezvous('download_only_once')  # Make sure only the first process in distributed training will download model & vocab
```

Comment:
I'm assuming no tensor work is done in the methods b/w `download_only_once` rendezvous's

Reply:
Yep, correct.